### PR TITLE
Create graphs.py

### PR DESCRIPTION
**Pull request by rishabhpranav**
Graph being an important part of DS ALGO is added to this repository.

### CODE REVIEW

**Comment by codesankalp** [comment]
@rishabhpranav to get your pull request reviewed or to be valid please link your pull request with the issue.

**Comment by devkapilbansal** [comment]
@rishabhpranav it is required to be active for the time period till your pull request is accepted by maintainers

File: dsalgo/graphs.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Python3 Program to print BFS traversal 
+# from a given source vertex. BFS(int s) 
+# traverses vertices reachable from s.
```
Comment (devkapilbansal) [inline dsalgo/graphs.py:3]: Maybe we don't need these comments above import lines. What do you think @codesankalp

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@rishabhpranav Please see this review

**Comment by devkapilbansal** [comment]
Closing because of inactivity

**Pull request closed without merging**